datadog_lambda/tracing.py

```diff
@@ -685,9 +685,32 @@ def create_inferred_span(
     return None
 
 
+def get_service_mapping(service_name):
```

Comment:
```suggestion
service_mapping = {}
for entry in os.getenv("DD_SERVICE_MAPPING", "").split(","):
    parts = entry.split(":")
    if len(parts) == 2:
        key, value = parts
        service_mapping[key.strip()] = value.strip()
```

Reply:
I wonder if making this a global variable has any implications in production? basically if we make it a global variable then that service_mapping dict will only get assigned once the whole tracing.py module/file is loaded.

I know it broke my tests because the DD_SERVICE_MAPPING env var isn't being parsed for each test, only for the 1st one

Reply:
yeah I think its best to keep this as a function

Reply:
or else it becomes hard to test without reloading the tracing.py module